packages/caip/package.json

```diff
@@ -16,7 +16,7 @@
     "build": "yarn clean && yarn run -T tsc --build",
     "clean": "rm -rf dist tsconfig.tsbuildinfo",
     "dev": "yarn run -T tsc --build --watch",
-    "generate": "yarn generate:coingecko && yarn generate:coincap && yarn generate:yearn && yarn generate:osmosis",
+    "generate": "yarn build && yarn generate:coingecko && yarn generate:coincap && yarn generate:yearn && yarn generate:osmosis",
```

Comment:
in hindsight, maybe we just add it to each generate script, `yarn build` doesn't take too long and will prevent same issue if we ever run a specific `generate` script